app/etl/item/content_quality_service.rb

```diff
@@ -42,4 +43,6 @@ def convert_results(response, content)
   def count_metric(response, metric_name)
     response.dig(metric_name, 'count') || 0
   end
+
+  class NoContentError < StandardError; end
```

Comment:
Not sure...  If response code is not `200` we don't know what the error is. I would not named it `NoContentError`. I'd rather rename it to `QualityMetricsError` (or similar), and I would add the error to the exception so we can track it. What do you think?